### ISSUE TITLE
Hyperlinks to headings

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#215]
<img width="660" alt="image" src="https://user-images.githubusercontent.com/2157566/149753154-ea5d86a8-30d2-41c1-8e25-10940c05a409.png">

@unode do you know how we can make section header "clickable" in a sense that there would be a URL pointing to it? I tried adding `<section id="...">` to it, but it did not seem to work...

I feel this should be the place where to add something: https://github.com/NEUBIAS/training-resources/blob/47cbbb25daa7cf892d0ef47e5320218256c1caf3/_layouts/module.html#L124

**Comment by unode** [comment]
If a field has an `id="idname"` and the ID is unique in the entire page (required for HTML to validate), you can produce a URL to that section by adding `#idname` to the end of the URL.

**Comment by tischi** [comment]
OK, I tried but that does not make it "clickable" such that it generates the URL, should it?

**Comment by unode** [comment]
no, it doesn't make it clickable. For that you need to have a `<a href=''>...</a>`.
Though if you want to make that more stylized you may want to look into changing the template.
